src/components/Display.js

```diff
@@ -1,9 +1,12 @@
+import PropTypes from 'prop-types';
 import './display.css';
 
-function Display() {
+function Display({ result }) {
   return (
-    <p className="display">0</p>
+    <p className="display">{result}</p>
   );
 }
```

Comment:
- [ ] Ouch! this is close but it seems like the display is not properly updating after the operator keys are pressed and it's only showing the initial input this leads to a bad UX and might confuse users so I'd suggest you fix it so that each key is properly rendered in the UI when pressed.
- Here's how it behaves:
  ![firefox_VwyKOyz4Fu](https://github.com/MostafaShahat2021/math-magician-SPA/assets/104968236/7b3e03e4-60c6-4ee6-93f2-f7d6c5bbc402)

Reply:
Hello @AbelG101, thanks for your suggestion, I'll try to fix that 🥲 .

Reply:
Hey @AbelG101 Could you please tell me how you made this snapshot gif, I'll be grateful